【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:03',     '1',     '14:29:03',     'T/401M01.DEF',     '14:29:03',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:03',     '2023/09/11 14:29:03' )
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:03',     '1',     '14:29:03',     'T/401M01.CSV',     '14:29:03',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:03',     '2023/09/11 14:29:03' )
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:03',     '1',     '14:29:03',     'T/402M01.DEF',     '14:29:03',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:03',     '2023/09/11 14:29:03' )
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:03',     '1',     '14:29:03',     'T/402M01.CSV',     '14:29:03',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:03',     '2023/09/11 14:29:03' )
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:30',     '1',     '14:29:30',     'T/401M01.DEF',     '14:29:30',     'NG',     'リモート サーバーがエラーを返しました: (530) ログインされていません',     '2023/09/11 14:29:30',     '2023/09/11 14:29:30' )
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:30',     '1',     '14:29:30',     'T/401M01.CSV',     '14:29:30',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:30',     '2023/09/11 14:29:30' )
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:30',     '1',     '14:29:30',     'T/402M01.DEF',     '14:29:30',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:30',     '2023/09/11 14:29:30' )
【Date】2023/09/11 14:29:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:29:31',     '1',     '14:29:31',     'T/402M01.CSV',     '14:29:31',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:29:31',     '2023/09/11 14:29:31' )
【Date】2023/09/11 14:29:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:29:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:32:00【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/11 14:32:00【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:32:00',     '1',     '14:32:00',     'T/401M01.DEF',     '14:32:00',     'NG',     'リモート サーバーがエラーを返しました: (530) ログインされていません',     '2023/09/11 14:32:00',     '2023/09/11 14:32:00' )
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:32:01',     '1',     '14:32:01',     'T/401M01.CSV',     '14:32:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:32:01',     '2023/09/11 14:32:01' )
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:32:01',     '1',     '14:32:01',     'T/402M01.DEF',     '14:32:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:32:01',     '2023/09/11 14:32:01' )
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:32:01',     '1',     '14:32:01',     'T/402M01.CSV',     '14:32:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:32:01',     '2023/09/11 14:32:01' )
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:32:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:35:23',     '1',     '14:35:23',     'T/401M01.DEF',     '14:35:23',     'NG',     'リモート サーバーがエラーを返しました: (530) ログインされていません',     '2023/09/11 14:35:23',     '2023/09/11 14:35:23' )
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:35:23',     '1',     '14:35:23',     'T/401M01.CSV',     '14:35:23',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:35:23',     '2023/09/11 14:35:23' )
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:35:23',     '1',     '14:35:23',     'T/402M01.DEF',     '14:35:23',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:35:23',     '2023/09/11 14:35:23' )
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:35:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:35:23',     '1',     '14:35:23',     'T/402M01.CSV',     '14:35:23',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:35:23',     '2023/09/11 14:35:23' )
【Date】2023/09/11 14:35:24【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:35:24【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:36:01',     '1',     '14:36:01',     'T/401M01.DEF',     '14:36:01',     'NG',     'リモート サーバーがエラーを返しました: (530) ログインされていません',     '2023/09/11 14:36:01',     '2023/09/11 14:36:01' )
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:36:01',     '1',     '14:36:01',     'T/401M01.CSV',     '14:36:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:36:01',     '2023/09/11 14:36:01' )
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:36:01',     '1',     '14:36:01',     'T/402M01.DEF',     '14:36:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:36:01',     '2023/09/11 14:36:01' )
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 14:36:01',     '1',     '14:36:01',     'T/402M01.CSV',     '14:36:01',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 14:36:01',     '2023/09/11 14:36:01' )
【Date】2023/09/11 14:36:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 14:36:02【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:14:38【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:14:38【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:14:38',     '1',     '15:14:38',     'T/401M01.DEF',     '15:14:38',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:14:38',     '2023/09/11 15:14:38' )
【Date】2023/09/11 15:14:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:14:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:14:40',     '1',     '15:14:40',     'T/401M01.CSV',     '15:14:40',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:14:40',     '2023/09/11 15:14:40' )
【Date】2023/09/11 15:14:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:14:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:14:42',     '1',     '15:14:42',     'T/402M01.DEF',     '15:14:42',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:14:42',     '2023/09/11 15:14:42' )
【Date】2023/09/11 15:14:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:14:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:14:44',     '1',     '15:14:44',     'T/402M01.CSV',     '15:14:44',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:14:44',     '2023/09/11 15:14:44' )
【Date】2023/09/11 15:15:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:15:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:15:01',     '1',     '15:15:01',     'T/401M01.DEF',     '15:15:01',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:15:01',     '2023/09/11 15:15:01' )
【Date】2023/09/11 15:15:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:15:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:15:03',     '1',     '15:15:03',     'T/401M01.CSV',     '15:15:03',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:15:03',     '2023/09/11 15:15:03' )
【Date】2023/09/11 15:15:05【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:15:05【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:15:05',     '1',     '15:15:05',     'T/402M01.DEF',     '15:15:05',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:15:05',     '2023/09/11 15:15:05' )
【Date】2023/09/11 15:15:07【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:15:07【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:15:07',     '1',     '15:15:07',     'T/402M01.CSV',     '15:15:07',     'NG',     'リモート サーバーに接続できません。',     '2023/09/11 15:15:07',     '2023/09/11 15:15:07' )
【Date】2023/09/11 15:15:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:15:11【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/11 15:32:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした
【Date】2023/09/11 15:32:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:32:58',     '1',     '15:32:58',     'T/401M01.DEF',     '15:32:58',     'NG',     'リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした',     '2023/09/11 15:32:58',     '2023/09/11 15:32:58' )
【Date】2023/09/11 15:32:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:32:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:32:58',     '1',     '15:32:58',     'T/401M01.CSV',     '15:32:58',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:32:58',     '2023/09/11 15:32:58' )
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:32:59',     '1',     '15:32:59',     'T/402M01.DEF',     '15:32:59',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:32:59',     '2023/09/11 15:32:59' )
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:32:59',     '1',     '15:32:59',     'T/402M01.CSV',     '15:32:59',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:32:59',     '2023/09/11 15:32:59' )
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:32:59【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:33:21',     '1',     '15:33:21',     'T/401M01.DEF',     '15:33:21',     'NG',     'リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした',     '2023/09/11 15:33:21',     '2023/09/11 15:33:21' )
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:33:21',     '1',     '15:33:21',     'T/401M01.CSV',     '15:33:21',     'NG',     'リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした',     '2023/09/11 15:33:21',     '2023/09/11 15:33:21' )
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:33:21',     '1',     '15:33:21',     'T/402M01.DEF',     '15:33:21',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:33:21',     '2023/09/11 15:33:21' )
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:33:21',     '1',     '15:33:21',     'T/402M01.CSV',     '15:33:21',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:33:21',     '2023/09/11 15:33:21' )
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:33:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:34:33',     '1',     '15:34:33',     'T/401M01.DEF',     '15:34:33',     'NG',     'リモート サーバーがエラーを返しました: (553) ファイル名が許可されませんでした',     '2023/09/11 15:34:33',     '2023/09/11 15:34:33' )
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:34:33',     '1',     '15:34:33',     'T/401M01.CSV',     '15:34:33',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:34:33',     '2023/09/11 15:34:33' )
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:34:33',     '1',     '15:34:33',     'T/402M01.DEF',     '15:34:33',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:34:33',     '2023/09/11 15:34:33' )
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:34:33',     '1',     '15:34:33',     'T/402M01.CSV',     '15:34:33',     'NG',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/11 15:34:33',     '2023/09/11 15:34:33' )
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:34:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:43:35【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:35',     '1',     '15:43:35',     '401M0001.DEF',     '15:43:35',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:35',     '2023/09/11 15:43:35' )
【Date】2023/09/11 15:43:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:40',     '1',     '15:43:40',     '401M0001.CSV',     '15:43:40',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:40',     '2023/09/11 15:43:40' )
【Date】2023/09/11 15:43:41【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:41',     '1',     '15:43:41',     '402M0001.DEF',     '15:43:41',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:41',     '2023/09/11 15:43:41' )
【Date】2023/09/11 15:43:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:44',     '1',     '15:43:44',     '402M0001.CSV',     '15:43:44',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:44',     '2023/09/11 15:43:44' )
【Date】2023/09/11 15:43:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:43:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:43:49【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:49',     '1',     '15:43:49',     '401M0001.DEF',     '15:43:49',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:49',     '2023/09/11 15:43:49' )
【Date】2023/09/11 15:43:50【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:50',     '1',     '15:43:50',     '401M0001.CSV',     '15:43:50',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:50',     '2023/09/11 15:43:50' )
【Date】2023/09/11 15:43:51【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:51',     '1',     '15:43:51',     '402M0001.DEF',     '15:43:51',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:51',     '2023/09/11 15:43:51' )
【Date】2023/09/11 15:43:52【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-11 15:43:52',     '1',     '15:43:52',     '402M0001.CSV',     '15:43:52',     'OK',     '226 226 Transfer complete.
',     '2023/09/11 15:43:52',     '2023/09/11 15:43:52' )
【Date】2023/09/11 15:43:52【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 15:43:52【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/11 17:19:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12 09:21:54',     '1',     '09:21:54',     '402M0001.DEF',     '2023-09-12 09:21:54',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:21:54',     '2023/09/12 9:21:54' )
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12 09:21:54',     '1',     '09:21:54',     '402M0001.CSV',     '2023-09-12 09:21:54',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:21:54',     '2023/09/12 9:21:54' )
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:21:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:30:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:30:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12',     '1',     '09:30:31',     '402M0001.DEF',     '2023-09-12 09:30:31.31 ',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:30:31',     '2023/09/12 9:30:31' )
【Date】2023/09/12 09:30:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:30:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:31:08【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:31:08【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12',     '1',     '09:31:08',     '402M0001.DEF',     '2023-09-12 09:31:08.08 ',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:31:08',     '2023/09/12 9:31:08' )
【Date】2023/09/12 09:31:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:31:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:36:06【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:36:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:36:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12',     '1',     '09:36:33',     '401M0001.CSV',     '2023-09-12 09:36:33.33 ',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:36:33',     '2023/09/12 9:36:33' )
【Date】2023/09/12 09:36:34【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/12 09:36:34【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】InsertTRNLOG【Description】SQL実行時エラー: INSERT INTO TRN_LOG(             PROCESS_DATE,             MACHINE_NO,             PROCESS_TIME,             FILE_NAME,             MASTER_SEND_TIME,             MASTER_RESULT,             NOTE,             CREATE_DATE,             UPDATE_DATE ) VALUES(     '2023-09-12',     '1',     '09:36:34',     '402M0001.CSV',     '2023-09-12 09:36:34.34 ',     'False',     'リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています',     '2023/09/12 9:36:34',     '2023/09/12 9:36:34' )
【Date】2023/09/12 09:36:34【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:43:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:43:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:43:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:43:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:43:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:48:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:48:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:48:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:28【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:28【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:28【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:28【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:50:29【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:51:43【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:51:43【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:51:43【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:51:43【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:51:43【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:52:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:52:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:52:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:52:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:52:40【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:56:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:56:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:56:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:56:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:56:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:57:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:57:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:57:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:57:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:58:41【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:58:41【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:58:41【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:58:41【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 09:58:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:01:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:01:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:01:15【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:01【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:02【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:02:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:08:39【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:08:39【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:08:39【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:09:48【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:09:48【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:11:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:26【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:27【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 10:12:27【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 11:57:09【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 11:57:14【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 11:57:16【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 11:57:19【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 15:12:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 15:12:37【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 15:15:03【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】ファイル 'C:\SANPAI_DX\FTP_UPLOAD\401M0001.DEF' が見つかりませんでした。
【Date】2023/09/13 15:15:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 15:15:22【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 15:15:23【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/13 15:17:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】ファイル 'C:\SANPAI_DX\FTP_UPLOAD\401M0001.DEF' が見つかりませんでした。
【Date】2023/09/13 15:19:36【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】ファイル 'C:\SANPAI_DX\FTP_UPLOAD\401M0001.DEF' が見つかりませんでした。
【Date】2023/09/13 15:26:46【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】ファイル 'C:\SANPAI_DX\FTP_UPLOAD\401M0001.DEF' が見つかりませんでした。
【Date】2023/09/13 15:28:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/13 15:29:00【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (530) ログインされていません
【Date】2023/09/14 12:07:27【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 12:07:28【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 12:07:36【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 12:07:37【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 13:07:38【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】パス 'C:\SANPAI_DX\FTP_BACKUP\UNIT01\401A0001.CSV' の一部が見つかりませんでした。
【Date】2023/09/14 13:07:39【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 13:07:47【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 13:07:48【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/14 13:40:48【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】パス 'C:\SANPAI_DX\FTP_BACKUP\UNIT01\401A0001.CSV' の一部が見つかりませんでした。
【Date】2023/09/14 13:40:49【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 13:40:53【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:01:06【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:05:47【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:06:56【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:07:58【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:10:35【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:12:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:12:32【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:15:32【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 14:15:44【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】AnsDownloadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/14 17:39:17【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/14 17:40:29【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/14 17:40:52【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/14 17:44:33【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/14 17:45:54【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 10:31:06【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:40:32【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:40:55【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:41:18【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:49:45【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:50:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:55:27【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:56:13【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 11:58:21【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 12:01:05【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 12:03:10【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 13:17:00【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 13:18:31【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 13:20:07【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 13:21:07【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/15 13:21:37【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーに接続できません。
【Date】2023/09/28 11:02:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/28 14:51:36【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (421) サービスが使用できません。管理接続を閉じています
【Date】2023/09/28 15:21:30【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
【Date】2023/09/28 15:21:42【Exe】ConsoleApp_Upload.exe【Source】Module_Upload【Method】UploadFtp【Description】リモート サーバーがエラーを返しました: (550) ファイルが使用できません (例: ファイルが見つからない、ファイルへのアクセスがない)
